project/scalapb.sbt

```diff
@@ -0,0 +1,3 @@
+libraryDependencies ++= Seq(
+  "com.github.os72" % "protoc-jar" % "3.0.0-b1"
```

Comment:
Any reason this is in a separate file, not `scio-schemas` section in `build.sbt`? This here applies to all sub-modules right?